.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+
+python:
+ - "3.7-dev"
+ - "3.6"
+ - "3.5"
+ - "3.4"
+ - "3.3"
+ - "3.2"
+ - "2.7"
+ - "2.6"
+
+script:
+ # Simple test runs
+ - ./local-install.sh
```

Comment:
local/remote could be put into env var and then reused in matrix.

Reply:
I'd prefer not to mix these things in a single job. Having them separate helps understand what fails better and faster.